protobuf/check_custom_value.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package Trento.Checks.V1;
+option go_package = "/;events";
+
+message CheckCustomValue {
+    string name = 1;
+    oneof value {
+        string string_value = 2;
+        int32 int_value = 3;
```

Comment:
`int32` because it seems `int64` might not be strictly necessary. But open to change it if we feel safer with a 64 bit integer.

Reply:
I think we could simply use `google.protobuf.Value`.
```
google.protobuf.Value value = 2;
```
It includes all the standard types and we already have the decoding

Reply:
you mean instead of the `oneof` just having
```go
google.protobuf.Value value = 2;
google.protobuf.Value original_value = 3;
```

I'd be fine with that, it's even simpler 😄, however wouldn't that be too many types?

Reply:
just out of convenience, whatever feels better to use. No hard opinions either way.
The 3 options in the oneOf is totally correct